usr/sbin/registercloudguest

```diff
@@ -577,63 +644,9 @@ def main(args):
 
     # Check if the target RMT for the registration is alive or if we can
     # find a server that is alive in this region
-    registration_target_found = False
-    if registration_smt:
-        registration_smt_cache_file_name = (
-            os.path.join(
-                utils.get_state_dir(),
-                utils.REGISTERED_SMT_SERVER_DATA_FILE_NAME
-            )
-        )
-        updated = utils.update_rmt_cert(registration_smt)
-        alive = registration_smt.is_responsive()
-        if alive:
-            msg = 'Instance is registered, and update server is reachable, '
-            msg += 'nothing to do'
-            # The cache data may have been cleared, write if necessary
-            if not os.path.exists(registration_smt_cache_file_name) or updated:
-                utils.store_smt_data(
-                    registration_smt_cache_file_name,
-                    registration_smt
-                )
-                if not utils.has_rmt_in_hosts(registration_smt):
-                    utils.clean_hosts_file(registration_smt.get_domain_name())
-                    utils.add_hosts_entry(registration_smt)
-            # The registry target may be missing, write if necessary
-            if not utils.has_registry_in_hosts(registration_smt):
-                utils.clean_hosts_file(registration_smt.get_domain_name())
-                utils.add_hosts_entry(registration_smt)
-            logging.info(msg)
-            registration_target_found = True
-        else:
-            # The current target server is not resposive, lets check if we can
-            # find another server
-            for new_target in region_smt_servers:
-                if (
-                        not new_target.is_responsive() or
-                        not registration_smt.is_equivalent(new_target)
-                ):
-                    continue
-                smt_ip = new_target.get_ipv4()
-                if utils.has_rmt_ipv6_access(new_target):
-                    smt_ip = new_target.get_ipv6()
-                msg = 'Configured update server is unresponsive, switching '
-                msg += 'to equivalent update server with ip %s' % smt_ip
-                utils.replace_hosts_entry(registration_smt, new_target)
-                utils.store_smt_data(
-                    registration_smt_cache_file_name,
-                    new_target
-                )
-                utils.update_rmt_cert(new_target)
-                registration_smt = new_target
-                registration_target_found = True
-                break
-            else:
-                msg = 'Configured update server is unresponsive. '
-                msg += 'Could not find a replacement update server '
-                msg += 'in this region. Possible network configuration issue'
-                logging.error(msg)
-                sys.exit(1)
+    registration_smt, reg_target_found = find_alive_registration_target(
```

Comment:
Can we keep the variable naming consistent and stick with `registration_target_found` instead of reg_target_found

it will also make the patch smaller because the subsequent `if registration_target_found` will then stay unchanged.

Thanks